dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/OAuth20EndpointServices.java

```diff
@@ -474,6 +477,23 @@ public void logout(OAuth20Provider provider,
         }
     }
 
+    String URLEncodeParams(String UrlStr) {
+        String sep = "?";
+        String encodedURL = UrlStr;
+        int index = UrlStr.indexOf(sep);
+        if (index > -1) {
+            index += 3;
```

Comment:
I'm not seeing where this `+3` comes into play; right now this falls under the category of a magic number. I could understand adding 1 to skip over the `?` character, but 3?

Reply:
fixed, thanks